### PR TITLE
Activate corepack

### PR DESCRIPTION
**Pull request by orhtej2**
## Problem

- Interactive prompt on `install`/`upgrade`

## Solution

- 2 step `pnpm` installation with proper path.

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by orhtej2** [comment]
!testme

**Comment by orhtej2** [comment]
!bookwormtestme

File: scripts/install [hunk 1]
```diff
@@ -60,8 +60,10 @@ pushd "$install_dir"
 	curl https://sh.rustup.rs -sSf | ynh_exec_warn_less ynh_exec_as $app sh -s -- -y
 	export PATH="$install_dir/.cargo/bin:$PATH"	
 	ynh_use_nodejs
-	corepack enable
-	corepack prepare pnpm@latest --activate
+	env $ynh_node_load_PATH corepack enable
+	env $ynh_node_load_PATH corepack prepare pnpm@latest --activate
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH corepack enable
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH corepack prepare pnpm@latest --activate
 	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH pnpm install
```
Comment (ericgaspar) [inline scripts/install]:
```suggestion
	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH corepack pnpm install
```

I was reading from the link below that `corepack` must be used along with `pnpm install`

cf -> https://www.raulmelo.me/en/blog/making-the-switch-to-pnpm

File: scripts/install [hunk 2]
```diff
@@ -60,8 +60,10 @@ pushd "$install_dir"
...
 	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_ENV=production pnpm build
```
Comment (ericgaspar) [inline scripts/install]:
```suggestion
	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_ENV=production corepack pnpm build
```

File: scripts/install [hunk 3]
```diff
@@ -60,8 +60,10 @@ pushd "$install_dir"
...
 	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH pnpm run init
```
Comment (ericgaspar) [inline scripts/install]:
```suggestion
	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH corepack pnpm run init
```

**Comment by ericgaspar** [comment]
!testme

**Comment by Thovi98** [comment]
!testme

**Pull request closed without merging**